artifact-evaluation/artifact.tex

```diff
@@ -116,10 +116,15 @@ \subsubsection{Software dependencies}
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 \subsection{Experiment workflow}
 
-Access the docker image \texttt{opencompl-ssa} from
-\url{https://zenodo.org/records/11519739}.
+Access the docker image \texttt{lean-mlir} from
+\textcolor{red}{Zenodo link}.
 
 \begin{minted}[fontsize=\scriptsize]{text}
+  from folder lean-mlir: 
+- docker build -f artifact-evaluation/Dockerfile -t lean-image .
+- docker create --name temp-container lean-image
+- docker cp temp-container:/code/lean-mlir/bv-evaluation/raw-data ./docker-results
+- docker rm temp-container
 $ docker load -i opencompl-ssa.tar
 $ docker run -it siddudruid/opencompl-ssa
```

Comment:
Why do we have `siddudruid` here? In fact, why do we change this file at all? Should we maybe name it `artifact-conference-`.

Reply:
done